### PR TITLE
Fix add troubleshooting link

### PR DESCRIPTION
**Pull request by sachscode**
#### Description of changes
Added link to troubleshooting guide url for  push, pull and publish errors.
The URL [https://docs.amplify.aws/project/troubleshooting/] is being added as part of 
PR https://github.com/aws-amplify/docs/pull/3960 and hence can only be merged once the documentation PR is merged.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/src/commands/push.ts [hunk 1]
```diff
@@ -16,7 +16,9 @@ import { collectDirectivesByTypeNames, readProjectConfiguration } from 'graphql-
 import path from 'path';
 import fs from 'fs-extra';
 
+
 const spinner = ora('');
+const TROUBLESHOOTING_PROJECT_URL = "https://docs.amplify.aws/project/troubleshooting/";
```
Comment (cjihrig) [inline packages/amplify-cli/src/commands/push.ts:21]: Why is this URL being duplicated in this file?

File: packages/amplify-cli/src/commands/help.ts [hunk 2]
```diff
@@ -1,5 +1,17 @@
+import { printer } from 'amplify-prompts';
 import { showAllHelp } from '../extensions/amplify-helpers/show-all-help';
 
+const TROUBLESHOOTING_PROJECT_URL = "https://docs.amplify.aws/project/troubleshooting/";
```
Comment (ammarkarachi) [inline packages/amplify-cli/src/commands/help.ts:4]: I think 'amplify-cli-core' is a better place to put this

Comment (sachscode) [reply packages/amplify-cli/src/commands/help.ts]: moved this to cli-core

File: packages/amplify-cli/src/commands/push.ts (hunk 1 above)
Comment (sachscode) [reply packages/amplify-cli/src/commands/push.ts]: moved this to cli-core

**Review by ammarkarachi — APPROVED** [review]

**Pull request merged**